process/sandbox-annual-review.md

```diff
@@ -0,0 +1,43 @@
+# Sandbox annual review 
+
+Sandbox projects are subject to an annual review by the TOC. This is intended to be a lightweight process to ensure that projects are on track, and getting the support they need.
+
+## How to file your annual review
+
+CNCF staff will notify the project maintainers when the project review is due. 
+
+Project maintainers are responsible for agreeing between them who will complete the annual review. One of the maintainers should create the review in GitHub under [cncf/toc/reviews](https://github.com/cncf/toc/tree/master/reviews). 
+
+* Raise a PR titled *[Project name] Annual Review*
+* The PR should include a file called `annual-<project name>-<year>.md` (for example, `annual-amazingproj-2019.md`) with the contents described below
+* Send an email to the TOC mailing list so that the community knows the PR is there and can comment on it
+
+If your annual review isn’t complete within two months of notification, we will take this as a sign that the project is not under active maintenance and the TOC is likely to decide to archive the project. 
```

Comment:
should this be changed to "If your annual review is not submitted within two months of notification"? What if the project has submitted for annual review before or after the notification is received, but the TOC or the CNCF SIG groups couldn't pick it up in time to do the review due to their backlog?

Reply:
Very good point, yes